core/src/main/java/org/apache/accumulo/fate/util/Retry.java

```diff
@@ -148,6 +158,7 @@ public long retriesCompleted() {
   public void waitForNextAttempt() throws InterruptedException {
     log.debug("Sleeping for {}ms before retrying operation", currentWait);
     sleep(currentWait);
+    waitIncrement *= waitFactor;
```

Comment:
I'm not sure how useful it would be, but we could add a tiny bit of jitter here with a small random number.

Reply:
Thank you for the fast review.    I think the jitter aspect is important and I will implement it.